tests/functional/pv/space_reclaim/test_staggered_cronjob_scheduling.py

```
@@ -328,3 +354,787 @@ def _assert_stagger_spread(self, timestamps: dict) -> None:
+        ocp_jobs = OCP(kind=constants.RECLAIMSPACEJOBS, namespace=pvc_namespace)
+        jobs_before = ocp_jobs.get().get("items", [])
+        jobs_before_count = len(
+            [
+                j
+                for j in jobs_before
+                if j["metadata"]["name"].startswith(f"{cj_obj.resource_name}-")
+            ]
+        )
+
+        logger.info("Resuming CronJob '%s'", cj_obj.resource_name)
+        resume_time = datetime.utcnow()
```

Ruff (0.15.13)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)